### PR TITLE
Use hyperkube binary, fix kraken local

### PR DESCRIPTION
**Pull request by spiffxp**
See individual commits for details

### CODE REVIEW

File: cloud_configs/master.yaml.erb [hunk 1]
```diff
@@ -210,23 +210,34 @@ coreos:
...
+        Requires=install-hyperkube.service
         After=wait4-local-etcd.service
 
         [Service]
         LimitNOFILE=infinity
-        ExecStartPre=-/usr/bin/mkdir -p /opt/bin
-        ExecStartPre=/usr/bin/wget -N -P /opt/bin https://storage.googleapis.com/kubernetes-release/release/<%= kubernetes_release %>/bin/linux/amd64/kube-apiserver
-        ExecStartPre=/usr/bin/chmod +x /opt/bin/kube-apiserver
-        ExecStart=/opt/bin/kube-apiserver \
+        ExecStart=/opt/bin/hyperkube apiserver \
```
Comment (mikeln) [inline cloud_configs/master.yaml.erb]: so, the "recommended" way is to start kubelet and provide the multi-node manifest file (either included or external).  Which starts a stand-alone kubelet which then spawns the other kubernetes processes in the manifest file:
[default file has:  apiserver, controller-manager, scheduler] vs what I see here which is discretely starting each piece (though it should still work).

Comment (mikeln) [inline cloud_configs/master.yaml.erb]: Actually, does this work?   Hyperkube is a docker image...not an executable.  It is meant to be an run via: docker run hyperkube <kub binary> <args>

Comment (spiffxp) [inline cloud_configs/master.yaml.erb]: I'm running it right now, it definitely works.  Hyperkube is the binary, all the other stuff about running it in a docker container and having kubelet orchestrate that is orthogonal.

Comment (mikeln) [inline cloud_configs/master.yaml.erb]: Ok, I only run the docker image version....I guess it got built from that exe :)
